languages/javascript/exercises/concept/arrays-analysis/.docs/introduction.md

```diff
@@ -0,0 +1,9 @@
+In Javascript, an array is a list-like structure with no fixed length which can hold any type of primitives or ojects, even mixed types. The array elements can be accesed by their index, array also provide a bunch of built-in methods. in this exercise we will tackle the build-in method to analyse the array
+
+Example of analysis without looping through it:
```

Comment:
Let's give the example using a `for` loop and _then_ the example using `indexOf`.
Index of still _loops_, but it's easier to _comprehend_ --